haystack/nodes/answer_generator/openai.py

```diff
@@ -56,12 +56,13 @@ def __init__(
         max_tokens: int = 13,
         top_k: int = 5,
         temperature: float = 0.2,
-        presence_penalty: float = -2.0,
-        frequency_penalty: float = -2.0,
+        presence_penalty: float = 0.1,
+        frequency_penalty: float = 0.1,
```

Comment:
We need some warning / info about this in the next release notes. Please make sure to include it there.